src/middleware.js

```diff
@@ -2,189 +2,200 @@ import RSAA from './RSAA';
 import { isRSAA, validateRSAA } from './validation';
 import { InvalidRSAA, RequestError } from './errors';
 import { normalizeTypeDescriptors, actionWith } from './util';
+import defaults from './defaults';
 
-/**
- * A Redux middleware that processes RSAA actions.
- *
- * @type {ReduxMiddleware}
- * @access public
- */
-function apiMiddleware({ getState }) {
-  return next => action => {
-    // Do not process actions without an [RSAA] property
-    if (!isRSAA(action)) {
-      return next(action);
-    }
+function createMiddleware(options = {}) {
+  const middlewareOptions = Object.assign({}, defaults, options);
```

Comment:
Thanks for the PR! I've only skimmed over this but have one immediate concern -- the 'responseOk' config seems much more useful on a per-request basis (ie, a property on `callAPI` below) than on the middleware as a whole.

Perhaps I'm misunderstanding the implementation here, but what are your thoughts? In either case, we'll need some docs explaining how to use this 😆